src/buildercore/core.py

```diff
@@ -599,6 +630,13 @@ def steady_stack_names(region):
     "convenience. returns names of all stacks in a non-transitory state"
     return stack_names(steady_aws_stacks(region))
 
+def adhoc_stack_names(region):
```

Comment:
rm this. 

logic in report can move 'up' into buildercore once we're happy with it.